opendatahub-io/odh-dashboard
# Update Node.js to v18 and npm to v9

## Description

Closes: #915 

## How Has This Been Tested?



1. Run the installation locally
2. Create a new build
3. Test that the build is working

Image: `quay.io/lferrnan/odh-dashboard:rhods-915`

**You should bump your local development to node 18 (nvm use 18) and npm v9**
**In some linux distributions npm +7 routes automatically with ipv6, so you might need to change your network config**

## Merge criteria:
